WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -59,6 +68,7 @@ public void onCreate(Bundle savedInstanceState) {
         }
 
         mPlugin = mPluginStore.getSitePluginByName(mSite, pluginName);
+        mPluginInfo = PluginUtils.getPluginInfo(mPluginStore, mPlugin);
```

Comment:
I think we should retrieve `mPlugInfo` *after* we check whether `mPlugin` is null.

Reply:
Fixed in 396ce521913f659432a59cc81f95fe147b9b6302